src/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wbpack Exercise</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" />
+</head>
+
+<body>
+    <main>
+        <div class="project-heading">
+            <p>Today's To DO</p>
+            <i class="fa-solid fa-rotate"></i>
+        </div>
+        <div class="input-field">
+            <input class="input-text" type="text" placeholder="to do list" />
+            <span><i class="fa-solid fa-turn-down-left"></i></span>
+        </div>
+        <div>
+            <ul class="new-task">
+                <!-- <li class="input-li">
+                    <label class="input-label" for="">
+							<input type="checkbox" />
+							<p>DO some</p>
+
+							<span class="menu-icon"
+								><i class="fa-solid fa-ellipsis-vertical"></i
+							></span>
+						</label>
+                </li> -->
```

Comment:
- [ Optional ] Kindly remove the commented codes from your code-base ,the commented code adds confusion with no real benefit and makes your code-base unclean .